udata/templates/organization/dashboard.html

```diff
@@ -14,14 +14,10 @@
 {% endblock %}
 
 {% block main_content %}
-<section>
-    <div class="container">
-        <dashboard-graphs object-id="{{ org.id }}"></dashboard-graphs>
-        <div class="col-sm-9" id="graphs" object-id="{{ org.id }}">
-            <div class="text-center"><span class="fa fa-4x fa-cog fa-spin"></span></div>
-        </div>
-    </div>
-</section>
+<dashboard-graphs object-id="{{ org.id }}">
+    {# Load spinner until JS is ready #}
+    <div class="text-center"><span class="fa fa-4x fa-cog fa-spin"></span></div>
```

Comment:
I guess a component does not make much sense here as it is waiting for JS to be ready and the component is a JS